plugins/url/reddit.go

```diff
@@ -45,37 +45,63 @@ type redditComment struct {
 	} `json:"data"`
 }
 
-var redditUserRegex = regexp.MustCompile(`^/(u|user)/([^/]+)$`)
+var redditUserRegex = regexp.MustCompile(`^/(?:u|user)/([^\s/]+)$`)
+var redditPrivmsgUserRegex = regexp.MustCompile(`(?:\s|^)/(?:u|user)/([^\s/]+)`)
 var redditCommentRegex = regexp.MustCompile(`^/r/[^/]+/comments/([^/]+)/.*$`)
-var redditSubRegex = regexp.MustCompile(`^/r/([^/]+)/?.*$`)
+var redditSubRegex = regexp.MustCompile(`^/r/([^\s/]+)/?.*$`)
+var redditPrivmsgSubRegex = regexp.MustCompile(`(?:\s|^)/r/([^\s/]+)`)
```

Comment:
I know you didn't add all of these, but can you add example strings for these regexes?